Bitcoin references removed

crowning- opened the pull request:
We should create https://github.com/dashpay/gitian.sigs/ one day :-)

UdjinM6 commented on doc/gitian-building.md:
```diff
@@ -363,11 +365,11 @@ and inputs.
-./bin/gbuild --commit bitcoin=${COMMIT} --url bitcoin=${URL} ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
-./bin/gbuild --commit bitcoin=${COMMIT} --url bitcoin=${URL} ../bitcoin/contrib/gitian-descriptors/gitian-win.yml
-./bin/gbuild --commit bitcoin=${COMMIT} --url bitcoin=${URL} ../bitcoin/contrib/gitian-descriptors/gitian-osx.yml
+URL=https://github.com/crowning-/dash.git
```
Line 381 should be changed also ;)

crowning- replied:
I removed laanwj.

UdjinM6 commented on doc/gitian-building.md:
```diff
@@ -394,5 +396,5 @@ Uploading signatures
 After building and signing you can push your signatures (both the `.assert` and `.assert.sig` files) to the
-[bitcoin/gitian.sigs](https://github.com/bitcoin/gitian.sigs/) repository, or if that's not possible create a pull
-request. You can also mail the files to me (laanwj@gmail.com) and I'll commit them.
+[dash/gitian.sigs](https://github.com/dashpay/gitian.sigs/) repository, or if that's not possible create a pull
```
We have no such repo.... Should we organize one @evan82 @schinzelh ?

schinzelh replied:
We once started storing the gitian.sigs in a Darkcoin repo, but never made to the point to use them to sign/verify the builds. I think it will be part of the improved release process

crowning- commented:
Removed laanwj :-) and added a workaround for a possible problem with make-base-vm

@schinzelh will hopeful have the gitian-sigs repository added to his ToDo list now, so from my point of view we can merge this one.

UdjinM6 commented:
I'm not sure.
For me it's looks like we are describing some process that doesn't actually exist but we have no note/disclaimer about it so it really looks like we are describing some actual process instead.
It's kind of a false statement (a lie) by us imo and could lead to unneeded confusion.
I see no benefit of fixing/actualizing some document this way actually.

crowning- commented:
I've changed the signature chapter to make clear that this is not yet available.

UdjinM6 commented:
Well, ok, let's merge ~~and forget about it~~ :trollface: 

@evan82 what do you think?

evan82 commented:
Dishonest? I don't think anything in the release process is dishonest. Currently I am the only one that signs the releases, but we've talked about moving to a more robust process where all of us use our own Gitian setup and each compile, then compare signatures. Currently, it seems to be a bit overkill, I think we should move to that when we are >100M in market cap. As for the other changes, everything looks good :+1:

UdjinM6 commented:
Ok, merging. Thanks everyone! :relaxed:
